src/transmission/processing/XTCEParser.py

```diff
@@ -94,3 +99,14 @@ class XTCEException(Exception):
     def __init__(self, message=""):
         self.message = message
         super().__init__(self.message)
+
+class SatParsers:
+
+    def __init__(self):
+        
+        self.parsers = {
+            "delfi_pq": XTCEParser("delfipq/Delfi-PQ.xml", "Radio"),
+            "delfi_next": None,
+            "delfi_c3": XTCEParser("delfic3/Delfi-C3.xml", "TLM"),
+            "da_vinci": None
+        }
```

Comment:
@StefanoSperetta do you have any pointers on how fix this error? I get it when running `python manage.py adddelfic3data` (same concept as with adddelfipqdata). It tries to process the data from `delfi-c3.txt` which came from Satnogs and there is only the TLM stream in the XML. I'm not sure what's off about it. 

```
2022-08-24 01:07:06,659 [process_raw_bucket | ERROR] delfi_c3: frame processing error: An error occurred while calling o1020.processStream.
: org.xtce.toolkit.XTCEDatabaseException: Unable to process binary for desired stream: 'TLM'
        at org.xtce.toolkit.XTCETMStream.processStream(XTCETMStream.java:224)
        at org.xtce.toolkit.XTCETMStream.processStream(XTCETMStream.java:254)
        at jdk.internal.reflect.GeneratedMethodAccessor17.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
        at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
        at py4j.Gateway.invoke(Gateway.java:282)
        at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
        at py4j.commands.CallCommand.execute(CallCommand.java:79)
        at py4j.GatewayConnection.run(GatewayConnection.java:238)
        at java.base/java.lang.Thread.run(Thread.java:829)
Traceback (most recent call last):
  File "/app/transmission/processing/XTCEParser.py", line 36, in processTMFrame
    model = self.stream_.processStream(data)
  File "/usr/local/lib/python3.9/site-packages/py4j/java_gateway.py", line 1321, in __call__
    return_value = get_return_value(
  File "/usr/local/lib/python3.9/site-packages/py4j/protocol.py", line 326, in get_return_value
    raise Py4JJavaError(
py4j.protocol.Py4JJavaError: An error occurred while calling o1020.processStream.
: org.xtce.toolkit.XTCEDatabaseException: Unable to process binary for desired stream: 'TLM'
        at org.xtce.toolkit.XTCETMStream.processStream(XTCETMStream.java:224)
        at org.xtce.toolkit.XTCETMStream.processStream(XTCETMStream.java:254)
        at jdk.internal.reflect.GeneratedMethodAccessor17.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
        at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
        at py4j.Gateway.invoke(Gateway.java:282)
        at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
        at py4j.commands.CallCommand.execute(CallCommand.java:79)
        at py4j.GatewayConnection.run(GatewayConnection.java:238)
        at java.base/java.lang.Thread.run(Thread.java:829)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/app/transmission/processing/process_raw_bucket.py", line 110, in process_raw_bucket
    store_frame(satellite, row["_time"], row["frame"], row["observer"],  link)
  File "/app/transmission/processing/process_raw_bucket.py", line 32, in store_frame
    telemetry = parser.processTMFrame(bytes.fromhex(frame))
  File "/app/transmission/processing/XTCEParser.py", line 49, in processTMFrame
    raise XTCEException(str(ex)) from ex
transmission.processing.XTCEParser.XTCEException: An error occurred while calling o1020.processStream.
: org.xtce.toolkit.XTCEDatabaseException: Unable to process binary for desired stream: 'TLM'
        at org.xtce.toolkit.XTCETMStream.processStream(XTCETMStream.java:224)
        at org.xtce.toolkit.XTCETMStream.processStream(XTCETMStream.java:254)
        at jdk.internal.reflect.GeneratedMethodAccessor17.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
        at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
        at py4j.Gateway.invoke(Gateway.java:282)
        at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
        at py4j.commands.CallCommand.execute(CallCommand.java:79)
        at py4j.GatewayConnection.run(GatewayConnection.java:238)
        at java.base/java.lang.Thread.run(Thread.java:829)
```